### DISCUSSION TITLE
Python 3.9 support

### DISCUSSION THREAD

**Discussion by shahyash10** [oneapi-src/oneDAL#2017] [Q&A] [answered]
When will support for python 3.9 be added?
Thanks!

**Comment by PetrovKP** [comment]
Hi @yashs97 
Planned in the next release  - 2021.4. Around the end of September.

**Comment by SmirnovEgorRu** [comment, marked as answer]
@yashs97, python 3.9 is supported in conda-forge and main conda channels already. In pip it will be supported in the next release as @PetrovKP mentioned.

```sh
conda install scikit-learn-intelex -c conda-forge / main
conda install daal4py -c conda-forge / main
```
